notes-with-indexeddb/notes.js

```diff
@@ -0,0 +1,169 @@
+/*
+Feature detection
+ */
+// DON'T use 'var indexedDB = ...' if you're not in a function.
+window.indexedDB = window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || window.msIndexedDB;
```

Comment:
No need for this. The bare `window.indexedDB` is supported anywhere this code is capable of running. Same for the other keys.